mdm/orientdb-api/src/main/java/com/gentics/mesh/core/data/job/JobRoot.java

```diff
@@ -76,6 +76,15 @@ public interface JobRoot extends RootVertex<Job> {
 	 */
 	HibJob enqueueVersionPurge(HibUser user, HibProject project);
 
+	/**
+	 * Enqueue a mail sending job.
+	 * @param user
+	 * @param now
+	 * @param mail
+	 * @return
+	 */
+	HibJob enqueueMailSending(HibUser user, long now, String mail);
```

Comment:
The API should also be changed accordingly, to accept distinct email parts.

Reply:
`HibJob enququeMailSending(HibUser user, long time, List<String> toAddrs, List<String> ccAddrs, List<String> bccAddrs, String title, String body, List<String> attachmentUuids)`

Reply:
`HibJob enququeMailSending(HibUser user, long time, MailSendingRequest request)`